views/module.pug

```diff
@@ -21,21 +21,18 @@ block content
           .form-group.col-sm
             label(for='locationCode') Side of Seawall
             select.form-control( id='locationCode' name='locationCode' required)
-              if mod.locationCode != ''
-                option !{mod.locationCode}
-              option E
-              option W
+              option(selected=mod.shape === 'E') E
```

Comment:
was the caching issue with these ever resolved? how does it work on the live herokuapp version

Reply:
I am not sure how or if one could  deal with it. Maybe there is a way to signal to the browser to reload but I am unsure